### PR TITLE
Update to React 0.14

### PR DESCRIPTION
**Pull request by paulathevalley**
- Ignored the npm-debug.log.* https://github.com/FormidableLabs/victory-pie/issues/9 
- Update to React 0.14 and victory-animation 0.0.8
 
/cc @boygirl

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,11 +60,11 @@
     "babel-core": "^5.5.8",
     "babel-loader": "^5.3.2",
     "d3": "^3.5.6",
-    "radium": "^0.13.4",
+    "radium": "^0.14.1",
     "rimraf": "^2.4.0",
     "style-loader": "~0.8.0",
     "url-loader": "~0.5.5",
-    "victory-animation": "latest",
+    "victory-animation": "^0.0.8",
```
Comment (boygirl) [inline package.json]: victory-animation@0.0.10 supports react 0.14

**Comment by boygirl** [comment]
@paulathevalley approved pending victory-animation version bump

**Comment by paulathevalley** [comment]
@boygirl Good to merge?

**Comment by boygirl** [comment]
:+1:

**Pull request merged**